extensions/control-plane/provision/gcs-provision/src/main/java/org/eclipse/dataspaceconnector/gcp/storage/provision/GcsProvisionExtension.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright (c) 2022 Google LLC
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Google LCC - Initial implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.gcp.storage.provision;
+
+import org.eclipse.dataspaceconnector.gcp.lib.iam.IamClientFactory;
+import org.eclipse.dataspaceconnector.gcp.lib.iam.IamService;
+import org.eclipse.dataspaceconnector.gcp.lib.iam.IamServiceImpl;
+import org.eclipse.dataspaceconnector.gcp.lib.storage.DefaultStorageClientFactory;
+import org.eclipse.dataspaceconnector.gcp.lib.storage.StorageClientFactory;
+import org.eclipse.dataspaceconnector.gcp.lib.storage.StorageService;
+import org.eclipse.dataspaceconnector.gcp.lib.storage.StorageServiceImpl;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.EdcSetting;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.Inject;
+import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.eclipse.dataspaceconnector.spi.transfer.provision.ProvisionManager;
+
+
+public class GcsProvisionExtension implements ServiceExtension {
+
+    @EdcSetting(required = true)
```

Comment:
please add values for the other attributes of the `EdcSetting`, as they will get picked up by our documentation generator